package.json

```diff
@@ -120,6 +120,7 @@
     "ts-node": "^9.0.0",
     "tsickle": "^0.39.0",
     "tslint": "^6.1.3",
+    "tslint-config-prettier": "^1.18.0",
```

Comment:
I just had a query on this one, and I'm a bit out of the loop on Angular eslint/tslint. This dependency looks like it's on the decline with no updates for a couple of years, but should the future plan be to remove tslint and therefore this dependency anyway? If this PR is purely concerned with Prettier then I think thats ok and we can defer the eslint one until later, just wanted to check with others (cc @elenagarrone)

Reply:
In my opinion this is fine for now as we will tackle eslint in a separate PR (hopefully soon).

Reply:
Yes I saw this is as more of a stop gap until we do the migration to ES Lint, in which case we'd need to swap this out for the ES Lint equivalent one - [eslint-plugin-prettier](https://github.com/prettier/eslint-plugin-prettier).